### PR TITLE
[Merged by Bors] - chore: Fix lemma name

### PR DESCRIPTION
**Pull request by YaelDillies**
Match https://github.com/leanprover-community/mathlib/pull/17967

### CODE REVIEW

File: Mathlib/Data/Int/Order/Basic.lean [hunk 1]
```diff
@@ -56,6 +56,8 @@ theorem abs_eq_natAbs : ∀ a : ℤ, |a| = natAbs a
   | -[_+1] => abs_of_nonpos <| le_of_lt <| negSucc_lt_zero _
 #align int.abs_eq_nat_abs Int.abs_eq_natAbs
 
+@[simp, norm_cast] lemma coe_natAbs (n : ℤ) : (n.natAbs : ℤ) = |n| := n.abs_eq_natAbs.symm
```
Comment (Vierkantor) [inline Mathlib/Data/Int/Order/Basic.lean:59]: Shouldn't this get an `#align`?
```suggestion
@[simp, norm_cast] lemma coe_natAbs (n : ℤ) : (n.natAbs : ℤ) = |n| := n.abs_eq_natAbs.symm
#align int.coe_nat_abs
```

Comment (YaelDillies) [reply Mathlib/Data/Int/Order/Basic.lean:59]: How does that work here? The lemma is not in mathlib yet.

Comment (Vierkantor) [reply Mathlib/Data/Int/Order/Basic.lean:59]: The two PRs should get merged basically simultaneously, right? And in any case `#align` itself doesn't seem to care about either of the declarations existing, only mathport.

**Comment by semorrison** [comment]
bors d+

File: Mathlib/Data/Int/Order/Basic.lean (hunk 1 above)
Comment (eric-wieser) [reply Mathlib/Data/Int/Order/Basic.lean:64]: @jcommelin  told me that #align fails if the mathlib4 definition doesn't exist; that certainly feels like it *ought* to be the case, but are you claiming it's not?

Comment (YaelDillies) [reply Mathlib/Data/Int/Order/Basic.lean:64]: Indeed it cares about the mathlib4 definition existing.

**Comment by YaelDillies** [comment]
bors merge

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**